ConservationInternational/resilienceatlas
# Journeys feedback

Journeys feedback
    
- [X] Show journey API slides on the home page
- [X] Fix chapter step image with filter
- [X] Fix view button position on map
-  <del>[ ] Fix map not centering properly on revisit  </del> - I can't reproduce this 🤷 
- [x] Remove Private Banner on embed maps on the Journeys
- [x]  Next Journey button is not working correctly
- [x] Some maps not displaying

## Testing instructions

Create a new journey and check the home page 

Also, see if the chapter steps are displayed correctly on the journeys

See that the view button on a map step is always displayed properly even if its the last step

Go to an embed map step and check that we don't see the Private Banner inside the embed

Test the next buttons on the steps. They should guide you through the journeys

## Tracking

https://vizzuality.atlassian.net/browse/RA2-24?atlOrigin=eyJpIjoiMDNjN2NjZmYxNjI3NGUzMmE3YmU3NGZjNzc3MzUwMjUiLCJwIjoiaiJ9
